### PR TITLE
readme: remove reference to old backup/restore

### PR DESCRIPTION
**Pull request by fanminshi**
[skip ci]
ref: #1626

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

**Comment by fanminshi** [comment]
PTAL cc/ @hongchaodeng

File: README.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ The etcd operator manages etcd clusters deployed to [Kubernetes][k8s-home] and a
 
 - [Create and destroy](#create-and-destroy-an-etcd-cluster)
 - [Resize](#resize-an-etcd-cluster)
-- [Recover a member](#member-recovery)
-- [Backup and restore a cluster](#disaster-recovery)
+- [Failure recovery](#failure-recovery)
 - [Rolling upgrade](#upgrade-an-etcd-cluster)
 
 There are [more spec examples](./doc/user/spec_examples.md) on setting up clusters with backup, restore, and other configurations.
```
Comment (hasbro17) [inline README.md]: Probably remove the backup and restore part of this to:
"There are [more spec examples](./doc/user/spec_examples.md) on setting up clusters with different configurations"

**Comment by hasbro17** [comment]
LGTM

**Pull request merged**